build.gradle

```diff
@@ -286,8 +286,8 @@ subprojects { subproject ->
         testImplementation group: 'org.powermock', name: 'powermock-api-mockito2', version: powermockVersion
         testImplementation group: 'org.powermock', name: 'powermock-module-junit4', version: powermockVersion
 
-        testImplementation group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.6'
-        implementation group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.6'
+        testImplementation group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.7-CCD-3694_CCD-3836'
+        implementation group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.7-CCD-3694_CCD-3836'
```

Comment:
Should be updated with proper release number once hmcts/befta-fw#192 is released.